ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetBucket.java

```diff
@@ -20,34 +20,45 @@
 import com.spectralogic.ds3cli.Arguments;
 import com.spectralogic.ds3cli.View;
 import com.spectralogic.ds3cli.ViewType;
+import com.spectralogic.ds3cli.exceptions.BadArgumentException;
 import com.spectralogic.ds3cli.exceptions.CommandException;
 import com.spectralogic.ds3cli.models.GetBucketResult;
 import com.spectralogic.ds3cli.views.cli.GetBucketView;
 import com.spectralogic.ds3cli.views.json.DataView;
+import com.spectralogic.ds3client.commands.spectrads3.GetBucketSpectraS3Request;
+import com.spectralogic.ds3client.commands.spectrads3.GetBucketSpectraS3Response;
 import com.spectralogic.ds3client.helpers.Ds3ClientHelpers;
+import com.spectralogic.ds3client.models.Bucket;
 import com.spectralogic.ds3client.models.Contents;
 import com.spectralogic.ds3client.networking.FailedRequestException;
+import com.spectralogic.ds3client.utils.Guard;
 import org.apache.commons.cli.Option;
 
 import static com.spectralogic.ds3cli.ArgumentFactory.BUCKET;
 import static com.spectralogic.ds3cli.ArgumentFactory.PREFIX;
 
 public class GetBucket extends CliCommand<GetBucketResult> {
 
-    private final static ImmutableList<Option> requiredArgs = ImmutableList.of(BUCKET);
-    private final static ImmutableList<Option> optionalArgs = ImmutableList.of(PREFIX);
+    private final static ImmutableList<Option> optionalArgs = ImmutableList.of(PREFIX, BUCKET);
```

Comment:
Is the reason the `BUCKET` was moved into the optional list to improve the error message reported?